### DISCUSSION TITLE
unrelated but you have to checkout `octo.nvim`

### DISCUSSION THREAD

**Discussion by molleweide** [cljoly/telescope-git-repo.nvim#21] [General]
I have just found this plugin and it seems quite spectacular.

`octo.nvim`

It supports creating issues and reviewing PRs from within neovim.
I am going to try to use it to create an issue here.

**Comment by cljoly** [comment]
Thanks!
I knew octo.nvim already, but I haven’t given it a try yet.

By the way, in the interest of keeping the conversations focused on `telescope-repo.nvim`, would it be possible for you to contact me using one of the methods in https://cj.rs/contact please?

**Reply by molleweide** [reply]
Allright ofc!!
